examples/storm-hbase-examples/pom.xml

```diff
@@ -26,6 +26,11 @@
     </parent>
 
     <artifactId>storm-hbase-examples</artifactId>
+    
+    <properties>
+        <!-- Required downgrade by hbase-client 1.4.4 -->
```

Comment:
Just curious, how did you explore whether downgrade is required?

Reply:
It's been a while, so I might be misremembering, but I believe I used `mvn dependency:tree` and checked whether each module had any dependencies that used Guava less than version 21.0. According to the Guava README, that was the last version to remove a non-beta API (https://github.com/google/guava#important-warnings point 2). My understanding is that Guava versions are backwards compatible starting at 21.0.

The downgrade may not actually be required, but I don't want to do the testing required to figure that out. At some point someone will upgrade the direct dependency, and then we can remove the downgrade.

Reply:
Thanks for explaining! I would also think this would be our best try with limited resource.